README.md

```diff
@@ -151,7 +77,7 @@ sudo gpasswd -a $USER docker
 newgrp docker
 ```
```

Comment:
ubuntu@ip-172-31-19-117:~$ sudo apt-get install docker.io
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package docker.io is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'docker.io' has no installation candidate

Reply:
I can add an apt-get update as the first command.

Reply:
Updated the readme to have `apt-get update` as the first step